src/other_functions.jl

```diff
@@ -178,3 +179,70 @@ polynomials. For more details, see [`Base.isapprox`](@ref).
 function isapprox{T<:AbstractSeries,S<:AbstractSeries}(x::T, y::S; rtol::Real=rtoldefault(x,y), atol::Real=0, nans::Bool=false)
     x == y || (isfinite(x) && isfinite(y) && norm(x-y,1) <= atol + rtol*max(norm(x,1), norm(y,1))) || (nans && isnan(x) && isnan(y))
 end
+
+
+#taylor_expand function for Taylor1
+doc"""
+    taylor_expand(f,x0;order)
```

Comment:
Can you add white spaces? I mean `taylor_expand(f, x0; order)`.

As commented below, I am **not** in favor of changing the number of variables inside the function.